src/state/columns/saga.js

```diff
@@ -4,16 +4,20 @@ import {
 } from 'redux-saga/effects'
 
 import { formatSumUpValue } from 'utils/columns'
-import { updateStatus } from 'state/status/actions'
+import { updateStatus, updateErrorStatus } from 'state/status/actions'
 
 import types from './constants'
 
 function* showColumnSumUp({ payload }) {
-  if (payload) {
+  const sum = formatSumUpValue(payload)
```

Comment:
could add a check if `payload` exist or add a default value to it to avoid undesired passed values to `formatSumUpValue`

Reply:
Actually, this check was removed in order to display the `0` sum-up value correctly. 
Cause this payload has already been pre-checked (and can't be `null` or `undefined`) when this action dispatches. 
But the `default` value, in any case, will not be superfluous.